src/OpenTelemetry.Exporter.Zipkin/README.md

```diff
@@ -15,13 +15,16 @@ dotnet add package OpenTelemetry.Exporter.Zipkin
 
 ## Configuration
 
+You can enable the the `ZipkinExporter` with the `AddZipkinExporter()`
+[extension method](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterHelperExtensions.cs#L35).
+
 You can configure the `ZipkinExporter` through
 `ZipkinExporterOptions` properties:
 
 * `ServiceName`: Name of the service reporting telemetry. If the `Resource`
    associated with the telemetry has "service.name" defined, then it'll be
    preferred over this option.
-* `Endpoint`: URI address to receive telemetry.
+* `Endpoint`: URI address to receive telemetry (default 'https://zipkin-server-name:9411/api/v2/spans').
```

Comment:
The default is `https://localhost:9411/api/v2/spans`

Reply:
updated to localhost

Reply:
also updated to HTTP to match implementatoin default